setup.py

```diff
@@ -56,7 +56,7 @@ def read(fname):
 
     # We don't declare our dependency on mxnet here because we build with
     # different packages for different variants (e.g. mxnet-mkl and mxnet-cu90).
-    install_requires=['sagemaker-containers>=2.2.4', 'retrying==1.3.3'],
+    install_requires=['sagemaker-containers>=2.2.5', 'retrying==1.3.3'],
```

Comment:
why aren't we just using the latest version of sagemaker-containers?

Reply:
the `>=` allows for the latest version. The reason we need a lower bound is because SageMaker Containers 2.2.5 is the first one that supports a user-supplied `transform_fn`

Reply:
make sense.